Targets/Domain/Sources/Model/ArticleModel.swift

```diff
@@ -36,8 +36,25 @@ extension ArticleModel {
         
     }
     
-    public func update(_ item: Article) {
-        
+    public func update(_ item: Article) async {
+        do {
+            guard let data = try await api.uploadArticle(by: .init(
```

Comment:
```suggestion
            guard let data = await api.uploadArticle(by: .init(
```
uploadArticle 메소드가 throws 하고 있지 않아 try 제거 해도 될 것 같습니다.

Reply:
오우 그러네요 놓쳤는데 감사합니다